.tours/welcome-to-dreamhouse.tour

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://aka.ms/codetour-schema",
+  "title": "👋 Welcome to Dreamhouse",
+  "steps": [
+    {
+      "title": "Welcome",
+      "file": "force-app/main/default/lwc/propertySummary/propertySummary.html",
+      "line": 22,
+      "description": "Hello! Welcome to Dreamhouse (LWC) sample app. \n\nThis application incorporates Lightning web components and other Lightning Experience features to deliver a rich user experience that helps Dreamhouse, a fictitious real estate company, manage its properties and brokers using the Salesforce Platform.\n\nThis tour will show you around the repo at a high level. \n\nIn this file, you can see a simple `propertySummary` Lightning Web Component that displays a card with a short summary of a property (e.g. a house). Properties are stored as `Property__c` custom objects. In `propertySummary`, we use a [`lightning-record-form`](https://developer.salesforce.com/docs/component-library/bundle/lightning-record-form/documentation) base component to display the record information."
+    },
+    {
+      "title": "Property Filters",
+      "file": "force-app/main/default/lwc/propertyFilter/propertyFilter.html",
+      "line": 23,
+      "description": "Dreamhouse manages a lot of properties so they built a 'Property Explorer' page that provides a custom search experience.\n\nThis page is built with custom Lightning Web Components.\n\nIt allows brokers to:\n1. filter properties base on certain attribute values (keyword, price, number of bedrooms...)\n1. select properties from a list\n1. explore selected property details.\n\nYou are looking at the `propertyFilter` component that renders the property filters."
```

Comment:
You could add this explanation into the property explorer flexipage and then go into the details of propertyFilter maybe